### DISCUSSION TITLE
Documentation - Support different languages! 🌍

### DISCUSSION THREAD

**Discussion by cryinkfly** [cryinkfly/Autodesk-Fusion-360-for-Linux#86] [Q&A]
Can anybody help me to translate my documentation into different languages? 😊

The following languages are available to use in Fusion 360: 

- German
- English
- Spanish
- French
- Italian
- Japanese
- Korean
- Chinese (PRC)

I would like to thank you in advance for your active support! ❤️

**Comment by madhead** [comment]
Do you want a Russian version? Do you already have any ideas on where to put the translations? I mean: a different branch, files with language suffixes, directories named by language?

**Reply by cryinkfly** [reply]
@madhead Yes, I want to translate the documentation into other languages and also Autodesk Fusion 360 that this program is available for more people in the world! 🌍

So that this goes even faster in the future, I have created a development area:

- https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/tree/main/files/extensions/development-branch/locale-extensions

However, I have not yet dealt with the creation of such extensions in detail ... 😅

**Reply by madhead** [reply]
@cryinkfly, check this out: https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/pull/136

Let me know if you want more files to be translated or another layout or whatever. I'll update the PR.

**Comment by fliuzzi02** [comment]
Hi, i would be glad to help with the italian translation!!! However i'm no expert in using Git and its features. Big ups for this great project!

**Reply by cryinkfly** [reply]
@fliuzzi02 I am glad to read that you want to help me with the translation. :+1:

**Reply by fliuzzi02** [reply]
[README -it_IT.md](https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/files/7775788/README.-it_IT.md)

Here is the translated file

**Reply by cryinkfly** [reply]
@fliuzzi02 Thank you for your help. :+1:

**Comment by TheRealAgentP** [comment]
If you need a Portuguese (Portugal) translation, I can try and help out there

**Reply by cryinkfly** [reply]
@TroubledThinker Then simply fork my project and translate the readme's etc. as you feel like it and have time! Thank you very much for your help!!!
